docs/source/support/troubleshooting.rst

```diff
@@ -0,0 +1,84 @@
+===============
+Troubleshooting
+===============
+
+
+self.InsufficientTokens(f"Insufficient token balance ({self.token_agent})
+-------------------------------------------------------------------------
+
+There is an insufficient amount of NU in your staking account.
+
+
+ValueError: {'code': -32000, 'message': 'no suitable peers available'}
+----------------------------------------------------------------------
+
+This is a geth error and not related to ``nucypher``. You are probably running a light node that needs
+full nodes willing to serve information to it, but no such nodes were found.
+
+
+nucypher.blockchain.eth.interfaces.ConnectionFailed: Connection Failed, is IPC enabled
+--------------------------------------------------------------------------------------
+
+It’s possible that the provider URI is incorrect.
+
+For example: if the scheme is ``ipc://`` then the path should be appended as is to the end ``ipc:///home/…``
+i.e. **THREE** slashes not two.
+
+
+nucypher.config.base.OldVersion: Configuration file is the wrong version. Expected version 1; Got version UNKNOWN_VERSION
+-------------------------------------------------------------------------------------------------------------------------
+
+This is an issue with upgrading from a previous version while still having an older configuration on-disk.
```

Comment:
What's the resolution here?

Reply:
This is the same problem that `chris_remus` is having 😄

Reply:
Is this perhaps too specific of a case to be included?

Reply:
Yes, I vote "too specific".

Reply:
✔️